docs/develop-resources/api-conventions/validations.md

```diff
@@ -57,8 +57,18 @@ type CloudBuildWorkerPoolSpec struct {
 
 Validates the immutability in the controller runtime, by comparing the difference between `spec` and `status`, together with the corresponding proto field "IMMUTABLE" **field_behavior**. (default behavior if using the direct auto-generated code)
 
-To see if you can use this approach, check if the resource proto supports **field_behavior** (See [AIP 203](https://google.aip.dev/203)). 
-If so, you just need to make sure
+All google services are located in [googleapis Github repo](https://github.com/googleapis/googleapis/tree/master/google),
+refer to your resource's API documentation to identify the service name, for example: [compute](https://cloud.google.com/compute/docs/reference/rest/v1).
+Once you identify the service, find the proper path to the proto files, for example:
```

Comment:
Non-blocker: Can we have a separate doc about API proto? This seems to be mentioned in mockGCP and here so the information seems to be duplicate.

Reply:
Right, I think it's necessary to add it here because contributor may not check the mockGCP doc.

Reply:
Agreed that mentioning of API proto information here is beneficial. Though I was thinking we may want to avoid repeating the information, but instead, split out the knowledge into a separate doc, and add a link to the doc in both mockGCP and this doc.

Reply:
"All google services are located in googleapis Github repo" --> This may be misleading. Could you help improving this section to show how to generate the proto from the open api?